data/saas/config/datadog_config.yml

```diff
@@ -0,0 +1,59 @@
+saas_config:
+  - fides_key: datadog_connector_example
+    name: Datadog SaaS Config
+    type: datadog
+    description: A sample schema representing the Datadog connector for Fidesops
+    version: 0.0.1
+
+    connector_params:
+      - name: domain
+      - name: api_key
+      - name: app_key
+      - name: page_limit
```

Comment:
I know it's not documented anywhere but let's change this to `page_size` to be consistent with the other connectors.

Reply:
ok.